### PR TITLE
Install notes (#255).

### PR DESCRIPTION
**Pull request by pq**
Fixes: https://github.com/flutter/flutter-intellij/issues/255

/cc @devoncarew @mit-mit @stevemessick  

Word-smithing welcome!

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+The Flutter Plugin is hosted in our own plugin repository.  Before it can be installed, the repository needs to be added to the IDE.
+
+1. Open plugin preferences (**Preferences>Plugins**)
+1. Select **”Browse repositories…”**
+1. In the **Browse Repositories** dialog, click **Manage repositories…”**
+1. In the resulting **Custom Plugin Repositories** pop-up, click the **+** button and paste `https://flutter.github.io/flutter-intellij/alpha/updatePlugins.xml` into the **Repository URL:** field and click **OK**
+1. In the **Browse Repositories** dialog, select **`flutter-intellij`** from the list of plugins (be sure it is not being filtered out of your results view by specifying **Category: All** or limiting it to the Flutter repository in the pull-down by the search box) and click the green **Install** button
+1. Once downloaded, click the **Restart IntelliJ IDEA** button
+
```
Comment (mit-mit) [inline docs/readme.md]: I suggest we add a small section on usage:

## Using the plugin

Please consult the [core user journeys wiki page](https://github.com/flutter/flutter-intellij/wiki/User-Journeys).

File: docs/readme.md [hunk 2]
```diff
@@ -1 +1,22 @@
-- placeholder -
+## Plugin Install Instructions
+
+
+### Supported IDEs
+
+
+The Flutter Plugin can be installed on following IntelliJ-based products:
```
Comment (mit-mit) [inline docs/readme.md]: Don't think `Plugin` should be capitalized?

File: docs/readme.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+* IntelliJ 2016.2+ ([Ultimate or Community](https://www.jetbrains.com/idea/download/))
+
+
+### Installation
```
Comment (mit-mit) [inline docs/readme.md]: We seem to be missing something. When I tested these in a clean VM, I got the following error:

```
Problems found loading plugins: Plugin "Flutter" was not loaded: required plugin "Dart" not installed.
```

**Comment by devoncarew** [comment]
If we're hosting on the github jekyll site, perhaps add a link to it from the main readme.md?

File: docs/readme.md (hunk 3 above)
Comment (pq) [inline docs/readme.md]: Ah right.  I was scoping this just to installation but it presupposes Dart is installed (and configured).  I'll inline those instructions.

File: docs/readme.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
+
+The Flutter Plugin is hosted in our own plugin repository.  Before it can be installed, the repository needs to be added to the IDE.
```
Comment (pq) [inline docs/readme.md]: Totally.  https://github.com/flutter/flutter-intellij/issues/264

**Comment by pq** [comment]
> If we're hosting on the github jekyll site, perhaps add a link to it from the main readme.md?

Yes! https://github.com/flutter/flutter-intellij/issues/264

File: docs/readme.md (hunk 2 above)
Comment (pq) [inline docs/readme.md]: Fixed.

File: docs/readme.md (hunk 3 above)
Comment (pq) [inline docs/readme.md]: OK.  There are some issues here.  I started to put together a flow to described Dart setup but hit #178.  I'd like to fix that then land another iteration here that has a cleaner flow.

**Comment by pq** [comment]
Addressed inline nits and added a few issues to track the next iteration.

@devoncarew PTAL

**Comment by devoncarew** [comment]
👍 lgtm

**Pull request merged**